sdcm/nemesis.py

```diff
@@ -4319,6 +4325,106 @@ def disrupt_add_remove_mv(self):
         finally:
             self.unset_current_running_nemesis(cql_query_executor_node)
 
+    def change_audit_config(self, audit_config: dict):
+        InfoEvent(f"Set audit configuration: {audit_config}").publish()
+        self.cluster.configuring_audit(**audit_config)
+        actual_audit_config = self.target_node.audit_config()
+        if audit_config != actual_audit_config:
+            raise ValueError(
+                f"Expected audit configuration is {audit_config}, but actually it is {actual_audit_config}")
+
+    @staticmethod
+    def get_current_time_as_timeuuid(session):
+        result = session.execute("select now() as now_timeuuid from audit.audit_log limit 1")
+        return result.one().now_timeuuid
+
+    def validate_audit_rows(self, audit_log, session, keyspace_name, category, start_read_audit_log_from_event_time, operation=None,
+                            limit_rows=None):
+        self.log.debug("Read audit log from event time: %s", start_read_audit_log_from_event_time)
+        time.sleep(60)
+        try:
+            audit_log.audit_log_filter(session=session,
+                                       start_from_event_time=start_read_audit_log_from_event_time,
+                                       filter_columns=TableColumns(category=category,
+                                                                   operation=operation,
+                                                                   keyspace_name=keyspace_name),
+                                       limit_rows=limit_rows)
+        except NotFoundMatchedRowsInAuditLog as error:
+            return str(error)
+
+        return ""
+
+    def disrupt_toggle_auditing_state(self):
```

Comment:
toggling suggests it would changed from enabled to disable, and also from disabled to enabled

I would expect from a toggle nemesis to make a change and leave it, until the next run of it.

we should be clear if this a nemesis, the revert the configuration, or leave it in place when it done.